src/pattern-library/components/Library.js

```diff
@@ -3,12 +3,13 @@ import { toChildArray } from 'preact';
 import { useState } from 'preact/hooks';
 
 import { Frame } from '../../components/containers';
-import { LabeledButton } from '../../components/buttons';
 
 import { jsxToString } from '../util/jsx-to-string';
 
 /**
  * @typedef LibraryBaseProps
+ * @prop {import("preact").ComponentChildren|string} [intro] - Optional
```

Comment:
This prop addition is the only change to any of the exported components' APIs here. The rest of the changes in this module are styling and extracting internal subcomponents; an in-depth review is not required unless you're into it.